### ISSUE TITLE
Feedback: Remindoro with saved text title 

### ISSUE THREAD

**Issue by Jackymancs4** [palerdot/remindoro#2]
When using the `Save Text to Remindoro` context menu, a remindoro with the title `undefined - undefined`.

I think that the title a localizable filler text like just `Undefined`, or even better an automatic generated title like `Text from $URL`

**Comment by palerdot** [comment]
Can you give more info like browser details? Is it present in both `firefox` and `chrome`?

**Comment by Jackymancs4** [comment]
Like for #3 , the problem arise only in Firefox.
Confronting the permissions in the manifests

https://github.com/palerdot/remindoro/blob/043391296816a38b7d59248215f030dee7ea3f97/app/manifests/firefox/manifest.json#L14-L21

https://github.com/palerdot/remindoro/blob/043391296816a38b7d59248215f030dee7ea3f97/app/manifests/chrome/manifest.json#L15-L26

It seems like Firefox lacks the permission `tabs` (or `activeTabs`, which I think would be better) that allows it to access tabs details like the url.

Every time it tries to create a remindoro containing such information (the url) it fails with 
```
ReferenceError: to_add is not defined
events-modular.js:139:10
```

**Comment by palerdot** [comment]
This issue is fixed in latest `0.2.3` version. Thank you for bringing this to notice ...
